youths/schema.py

```diff
@@ -0,0 +1,90 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id"]
+
+
+PreferredLanguage = graphene.Enum(
+    "PreferredLanguage", [(l[0].upper(), l[0]) for l in LANGUAGES]
```

Comment:
the indexes are probably out of order 🙃

Reply:
Forgot to discuss this with you. It does not work if the label is set to 1 as there is non ascii caracters.

Reply:
but here, you can actually go with the previous approach:
```python
PreferredLanguage = graphene.Enum(
    "PreferredLanguage", [(l[1].upper(), l[0]) for l in LANGUAGES]
)
```

Reply:
languages' labels work fine against the regex in graphene

Reply:
Venäjä gives error here. But this moves to another pr.